src/components/Calculator.js

```diff
@@ -1,15 +1,32 @@
+import { useState } from 'react';
 import Button from './Buttons';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 const Calculator = () => {
+  const [result, setResult] = useState({ total: null, next: null, operation: null });
+  const { total, next, operation } = result;
+
+  const handleInput = (e) => {
+    const value = e.target.innerHTML;
+    setResult(calculate(result, value));
+  };
+
   const btnValues = ['AC', '+/-', '%', '\u00F7', '7', '8', '9',
     'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
   return (
     <div className="container">
-      <div className="output-result">0</div>
+      <div className="output-result">{ total || next || operation || 0 }</div>
       <div className="keys-container">
         <div className="calc-key">
-          { btnValues.map((value) => (<Button key={value} value={value} />))}
+          { btnValues.map((value) => (
+            <Button
+              key={value}
+              value={value}
+              handleInput={handleInput}
+            />
+          )) }
+          ;
```

Comment:
- [ ] I just recognized when you are performing an operation let us say `20` - `10` it is only displaying 20 on the screen and then the response we need to display each button pressed in this case when I press `20` it should be displayed then press `-` display it then `10` it means each button pressed should be displayed
<img width="788" alt="Screenshot 2023-07-04 at 17 02 39" src="https://github.com/marvin-nyalik/mathmagicians/assets/44745855/a02c81f5-37bd-4c75-832e-41031a0da213">